src/common/conf_file.h

```diff
@@ -125,6 +125,8 @@ class ConfFile {
     static int parse_stdstring(const char *val, size_t val_len, void *storage, size_t storage_len);
     static int parse_uint8_vector(const char *val, size_t val_len, void *storage,
                                   size_t storage_len);
+    static int parse_uint32_vector(const char *val, size_t val_len, void *storage,
+                                   size_t storage_len);
```

Comment:
please split the addition of parse_uint32_vector() to a separate commit

Reply:
done